ide/bugtracking.commons/src/org/netbeans/modules/bugtracking/issuetable/NodeTableModel.java

```diff
@@ -175,10 +175,10 @@ public void setProperties(ColumnDescriptor[] props) {
 
                     Object o = props[i].getValue(ATTR_ORDER_NUMBER);
 
-                    if (o instanceof Integer) {
-                        sort.put(new Double(((Integer) o).doubleValue()), Integer.valueOf(ia));
+                    if ((o != null) && o instanceof Integer) {
+                        sort.put(Double.valueOf(((Integer) o).doubleValue()), ia);
```

Comment:
I would drop the `Double.valueOf`, the dupplicate type conversion (Integer -> double -> Double) looks stange. I would assume, that the VM autoboxes here and I think 

```java
sort.put(((Integer) o).doubleValue(), ia);
```

is more readable.

Reply:
Agreed..